specs/template.md

```diff
@@ -0,0 +1,118 @@
+# Spec Title
+
+## Overview
+
+*The name of the component, along with a high-level description.*
+
+### Background
+
+*Relevant historical or background information, related existing issues, etc.*
+
+### Use Cases
+
+*Primary use cases for this component.*
+  
+### Features
+
+*A list of the key features unique to this component.*
+
+### Risks and Challenges
+
+*Notable risks or challenges associated with implementing the component. Would we need to make any breaking changes to other parts of fast-dna in order to achieve this component's goals?*
+
+### Prior Art/Examples
+
+*Screenshots and/or links to existing, canonical, or exemplary implementations of the component.*
+
+---
+
+## Design
+
+*Describe the design of the component, thinking through several perspectives:*
+
+- *A customer using the component on a web page.*
+- *A developer building an app with the component and interacting through HTML/CSS/JS.*
+- *A designer customizing the component.*
+
+### API
+
+*The key elements of the component's public API surface:*
+
+- *Component Name*
+- *Props/Attrs*
+- *Methods*
+- *Events*
+
+*Consider high and low-level APIs. Attempt to design a powerful and extensible low-level API with a higher-level API for developer/designer ergonomics and simplicity.*
```

Comment:
The use of "high" and "low" are used, but, then you mention "higher-level".  Curious, if this should be "high-level" as well?  or is the intent that it could just be higher than the low.